### PR TITLE
Added kubevela reference to appmod and changed job for codebuild

### PR DESCRIPTION
**Pull request by hmuthusamy**
*Issue #, if available:*

*Description of changes:*
1. Defaulted the namespace to flux-system and changed the repourl to DNS
2. Added kubevela repo to appmodbloueprints

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by shapirov103 — APPROVED** [review]
LGTM

File: platform/infra/terraform/deploy-apps/manifests/kubevela-dev.yaml [hunk 1]
```diff
@@ -5,20 +5,15 @@ metadata:
...
-    targetRevision: 1.9.11
-    helm:
-      releaseName: kubevela-dev
-      values: |
-        multicluster:
-          enabled: false
   destination:
-    name: 'dev-cluster'
     namespace: vela-system
+    name: dev-cluster
```
Comment (elamaran11) [inline platform/infra/terraform/deploy-apps/manifests/kubevela-dev.yaml:10]: Do we need these changes for MGMT cluster.

**Review by elamaran11 — COMMENTED** [review]
One comment. Rest looks good.

File: platform/infra/terraform/deploy-apps/manifests/kubevela-dev.yaml (hunk 1 above)
Comment (hmuthusamy) [reply platform/infra/terraform/deploy-apps/manifests/kubevela-dev.yaml:10]: Its already deployed the same version in management cluster using terraform referring to appmod blueprints

**Pull request merged**